TEST_COVERAGE.md

```
@@ -0,0 +1,116 @@
+| Library               | Tests | Stmts | Branches | Funcs | Lines | Status    |
+| --------------------- | ----- | ----- | -------- | ----- | ----- | --------- |
+| ast-guard             | 356   | 89.5% | 76.5%    | 89.7% | 90.4% | ✅ Good   |
+| vectoriadb            | 357   | 82.9% | 81.0%    | 80.3% | 82.5% | ✅ Good   |
+| plugins               | 101   | 78.2% | 60.7%    | 61.0% | 79.9% | ⚠️ Medium |
+| adapters              | 70    | 72.2% | 61.5%    | 74.0% | 72.2% | ⚠️ Medium |
+| enclave               | 168   | 71.6% | 56.3%    | 75.0% | 71.9% | ⚠️ Medium |
+| json-schema-to-zod-v3 | 59    | 70.5% | 54.2%    | 83.3% | 71.9% | ⚠️ Medium |
+| mcp-from-openapi      | 21    | 67.5% | 54.3%    | 78.6% | 68.2% | ⚠️ Medium |
+| sdk                   | 125   | 37.0% | 7.0%     | 17.5% | 38.1% | ❌ Low    |
+
+**Total Tests: 1,257 passed (1 skipped)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)